spec/graphql/mutations/needs/create_need_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'rails_helper'
+
+RSpec.describe 'Create a new need', type: :request do
+  describe 'happy path' do
+    before :each do
+      @query =
+              <<~GQL
+                mutation {
+                  createNeed(input:
+                    {
+                    title: "Cleanup our park"
+                    pointOfContact: "test@example.com"
+                    description: "I've noticed a lot of litter in the park lately. I'd like to do a cleanup day with the community."
+                    startTime: "2021-08-31 11:00:00 -600"
+                    endTime: "2021-08-31 12:00:00 -600"
+                    city: "Denver"
+                    state: "Colorado"
+                    zipCode: "80218"
+                    supportersNeeded: 15
+                  }
+                  )
+                  {
+                  need {
+                    id
+                    title
+                  }
+                  errors
+                }
+                }
+              GQL
+    end
+
+    it 'creates a new Need in the database if all required data is present' do
+      post '/graphql', params: {query: @query}
```

Comment:
this probably isn't necessary, but for the sake of being complete it might be a good idea have something that establishes that the object wasn't already in the data base. i mean, theoretically we know it wasn't in there because we didn't run code that would put it there. but still, it's good to not make that assumption, and instead include a little something that shows for sure that before we run the post request,  the thing that we are supposedly makeing wasn't already made. A little something like...

 expect(Need.last).to eq(nil)

this addition probably isn't necessary for the test, but it's nice to have if we want to add it.

Reply:
I'm not sure I follow your thoughts here - this would create a new need for every submission, so are you thinking we'd want to check across the other fields to make sure the need's attributes don't exactly match an existing need?